### ISSUE TITLE
Project not working!

### ISSUE THREAD

**Issue by MeerUzairWasHere** [adrianhajdin/threads#105]
I was following the tutorial everything was fine, once I got to creating communities... I get stuck... my communities didn't show up properly, I tried many things to solve it nothing happened... then I cloned to your so called "finished threads" and add my local.env file but same thing happened...nothing is working

**Comment by MeerUzairWasHere** [comment]
And this is not actually finished project

**Comment by JannickPepe** [comment]
Yeah my communities does not work either and i followed the video, i mean i can create community in the clerk google tab but when i write in a community it does not display it nor in the community menu tab. 

I dont get the date time either on the post that got created.

And i as well also tried to download the zip and place my env, but same result

**Comment by STEALTHREED** [comment]
Hi Adrian, isn't there support for this project?
